.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+before_install:
+    - git clone https://github.com/nroduit/Weasis.git
+    - cd Weasis 
+    - git checkout tags/3.0.4
+    - mvn install
```

Comment:
Should we first do mvn package in the install step and install under script?

Reply:
"install" does "package" and copy the weasis packages into the local repository, what's point to separate this two steps and move copy part to build of extension?

Reply:
yeah this is actually ok - I was thinking that because we do mvn install in both phases we are doing the same thing again but just realized that they are two different directories so this is good.